release-notes/0.8/0.8.md

```diff
@@ -0,0 +1,73 @@
+<!--
+* Copyright (c) 2017, 2018 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.8 release notes
+
+## Supported platforms
+
+JCK compliance status for **OpenJDK with Eclipse OpenJ9** binaries:
+
+- Java 9 compliant
+- Java 8 expected to be compliant
```

Comment:
The focus at Adopt has been on running the Java 8 JCKs.  As the "non-production" use banner is still up at Adopt and won't come down until binaries pass the JCKs, we can't claim compliance for either 8 or 9.

Reply:
Handled, as above.